js/damage.js

```diff
@@ -685,7 +686,6 @@ function getDamageResult(attacker, defender, move, field) {
 	var finalMod = chainMods(finalMods);
 
 	var damage = [];
-	var cumulatedStatDrops = 0;
```

Comment:
Did you mean for this line to be removed?

Reply:
Yup, it was some sort of leftover variable I had forgotten to remove when I redid the multi-turn damage calc logic.
Don't worry though, this variable is used nowhere so we can peacefully get rid of it.